### PR TITLE
Allow Cassandra schema builder to use credentials

### PR DESCRIPTION
**Pull request by PS-EGHornbostel**
Currently the Cassandra schema builder job only works on open Cassandra installations. This PR fixes this by modifying the script so that two environmental variables, CASSANDRA_USER and CASSANDRA_PASSWORD, could be passed in to supply these credentials.

Signed-off-by: E.G. Hornbostel <eg.hornbostel@pricespider.com>

## Which problem is this PR solving?
- Currently the Cassandra schema builder job only works on open Cassandra installations. This PR fixes this by modifying the script so that two environmental variables, CASSANDRA_USER and CASSANDRA_PASSWORD, could be passed in to supply these credentials.

## Short description of the changes
- Modify docker.sh script to allow for credentials to be assed in to cqlsh

### CODE REVIEW

**Comment by yurishkuro** [comment]
build failed
```
Generating the schema for the keyspace jaeger_v1_dc1 and datacenter dc1
/cassandra-schema/docker.sh: line 39: syntax error near unexpected token `else'
/cassandra-schema/docker.sh: line 39: `else'
The command "if [ "$CASSANDRA_INTEGRATION_TEST" == true ]; then bash ./scripts/travis/cassandra-integration-test.sh ; else echo 'skipping cassandra integration test'; fi" exited with 2.
```

File: plugin/storage/cassandra/schema/docker.sh [hunk 1]
```diff
@@ -31,4 +33,9 @@ done
 
 echo "Generating the schema for the keyspace ${KEYSPACE} and datacenter ${DATACENTER}"
 
-MODE="${MODE}" DATACENTER="${DATACENTER}" KEYSPACE="${KEYSPACE}" /cassandra-schema/create.sh "${TEMPLATE}" | ${CQLSH} ${CQLSH_SSL} ${CQLSH_HOST}
+
+if [ -z "$PASSWORD" ] then
```
Comment (yurishkuro) [inline plugin/storage/cassandra/schema/docker.sh]: missing semicolon before `then`

Comment (PS-EGHornbostel) [reply plugin/storage/cassandra/schema/docker.sh]: pushed fix

**Comment by pavolloffay** [comment]
thanks @PS-EGHornbostel

**Pull request merged**